opta/commands/apply.py

```diff
@@ -194,26 +195,17 @@ def _apply(
     else:
         raise Exception(f"Cannot handle upload config for cloud {layer.cloud}")
 
-    previous_config: Optional[StructuredConfig] = cloud_client.get_remote_config()
-    if previous_config is not None and "opta_version" in previous_config:
-        old_opta_version = previous_config["opta_version"]
-        if (
-            old_opta_version not in [DEV_VERSION, ""]
-            and VERSION not in [DEV_VERSION, ""]
-            and semver.VersionInfo.parse(VERSION.strip("v")).compare(
-                old_opta_version.strip("v")
-            )
-            < 0
-        ):
-            raise UserErrors(
-                f"ou're trying to run an older version of opta (last run was at {previous_config['date']} with version {old_opta_version}). Please update to the latest version and try again!"
-            )
+    existing_config: Optional[StructuredConfig] = cloud_client.get_remote_config()
+    if existing_config is not None and "opta_version" in existing_config:
```

Comment:
Let's move this whole block into _verify_semver?

Reply:
addressed